dolphin/processor/config.py

```diff
@@ -288,8 +289,35 @@ def get_kwargs_likelihood(self):
                     prior_param.extend(i)
                     kwargs_likelihood['prior_ps'].append(prior_param)
 
+        if 'lens_light_option' in self.settings and \
+                'prior_lens_light_ellip' in self.settings['lens_light_option']:
+            kwargs_likelihood['custom_logL_addition'] = \
+                self.custom_logL_addition
+
         return kwargs_likelihood
 
+    def custom_logL_addition(self, kwargs_lens=None, kwargs_source=None,
+                             kwargs_lens_light=None, kwargs_ps=None,
+                             kwargs_special=None,
+                             kwargs_extinction=None):
+        """
+        Impose a Gaussian prior on the NFW scale radius R_s based on
+         Gavazzi et al. (2007).
+        """
+
+        pa_light = ellipticity2phi_q(kwargs_lens[0]['e1'],
+                                     kwargs_lens[0]['e2'])[0] * 180 / np.pi
+        pa_mass = \
+            ellipticity2phi_q(kwargs_lens_light[0]['e1'],
+                              kwargs_lens_light[0]['e2'])[0] * 180 / np.pi
+
+        sigma = self.settings['lens_light_option']['prior_lens_light_ellip']
+
+        log_prior = -0.5 * (pa_light - pa_mass) ** 2 / sigma ** 2 - np.log(
```

Comment:
Would a top hat distribution with a Gaussian tail be more appropriate ?

Reply:
Also, I need to figure what if the eccentricity angle goes from -45 to 45 degrees